tools/docker/Dockerfile

```diff
@@ -47,12 +55,14 @@ RUN echo y | android update sdk --no-ui --all --filter platform-tools | grep 'pa
 RUN echo y | android update sdk --no-ui --all --filter android-26 | grep 'package installed'
 RUN echo y | android update sdk --no-ui --all --filter android-27 | grep 'package installed'
 RUN echo y | android update sdk --no-ui --all --filter android-28 | grep 'package installed'
+RUN echo y | android update sdk --no-ui --all --filter android-29 | grep 'package installed'
```

Comment:
We should clean that up at some point to create a smaller image and with that faster builds. In the past we always had to support the current and next SDK while we migrate over... but now we have a bunch of unused ones here.

In fact in our other repos we do not install any SDK manually anymore (and accept only the licenses) because the Gradle step later in this script will on itself figure out what needs to be downloaded nowadays.